modules/trimmomatic/trimmomatic.nf

```diff
@@ -0,0 +1,79 @@
+nextflow.enable.dsl = 2
+
+
+params.save_mode = 'copy'
+params.results_dir = "${params.outdir}/trimmomatic"
+params.should_publish = true
+
+
+process TRIMMOMATIC {
+    tag "${genomeName}"
+    publishDir params.results_dir, mode: params.save_mode, enabled: params.should_publish
+
+    input:
+    tuple val(genomeName), path(genomeReads)
+
+    output:
+    tuple val(genomeName), path("*_{R1,R2}.p.fastq.gz")
```

Comment:
@Mxrcon , notice that the output of this process is named with a `p`  - I think that we need to do some stub related testing atleast to make sure that the downstream MTBSeq related logic does not break as it has its own specific file names.